addons/crm_mail_client_extension/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'CRM Mail Client Extension',
+    'version': '1.0',
+    'category': 'Sales/CRM',
+    'sequence': 5,
+    'summary': 'Turn emails received in your mailbox into leads and log their content as internal notes.',
+    'description': "Turn emails received in your mailbox into leads and log their content as internal notes.",
+    'website': 'https://www.odoo.com/page/crm',
+    'depends': [
+        'web',
+        'crm',
+        'mail_client_extension',
+        'crm_iap_lead_enrich'
```

Comment:
Normally after moving iap.enrich.api done in previous commit we should be able to depend on iap only, isn't it ? Anyway purpose if possible is to depend on iap_crm only (bridge between iap and base crm module).

+ update migration script accordingly

Reply:
This is right, the crm_iap_lead_enrich module is not needed in the CRM extension, furthermore, since IAP is only used in the base module, we don't need anything related to IAP here either.

The only needed dependencies now are "crm" and "mail_client_extension".